packages/oui-datagrid/src/index.spec.js

```diff
@@ -636,6 +636,40 @@ describe("ouiDatagrid", () => {
             const secondDatagridRows = getRows(angular.element(datagridElements[1]));
             expect(secondDatagridRows.length).toEqual(secondDatagridSize);
         });
+
+        describe("Columns", () => {
+            it("should support dataset notation", () => {
+                const element = TestUtils.compileTemplate(`
+                        <oui-datagrid data-rows="$ctrl.rows">
+                            <oui-column data-property="firstName"
+                                data-sortable="asc"
+                                data-title="'First name'"></oui-column>
+                            <oui-column property="lastName"
+                                data-sortable
+                                data-title="'Last name'"></oui-column>
+                            <oui-action-menu>
+                                <oui-action-menu-item text="Action 1"></oui-action-menu-item>
+                                <oui-action-menu-item text="Action 2"></oui-action-menu-item>
+                            </oui-action-menu>
+                        </oui-datagrid>
+                    `, {
+                        rows: fakeData.slice(0, 5)
+                    }
+                );
+
+                // Check data-sortable and data-title
+                const headerCells = getHeaderRow(element).find("th");
+                expect(headerCells[0].innerHTML).toEqual("First name");
+                expect(headerCells[0].className).toContain("oui-datagrid__header_sortable-asc");
+                expect(headerCells[1].innerHTML).toEqual("Last name");
+                expect(headerCells[1].className).toContain("oui-datagrid__header_sortable");
+
+                // Check data-property
+                const firstRow = getRow(element, 0);
+                expect(getCell(firstRow, 0).text().trim()).toEqual("Ann");
```

Comment:
`fakeRow[0].firstName` instead of hardcoding `Ann`. It will be easier if you need to change the dataset later.

Reply:
Saw together, it OK because data is sorted. Ignore my comment.

Reply:
Ann is not the first record of the dataset.